src/evict.c

```diff
@@ -510,6 +510,8 @@ static unsigned long evictionTimeLimitUs() {
  *   EVICT_FAIL     - memory is over the limit, and there's nothing to evict
  * */
 int performEvictions(void) {
+    /* we don't go to update_metrics on purpose because it's as if we don't
+     * call this function */
```

Comment:
```suggestion
    /* Note, we don't goto update_metrics here because this check skips eviction
     * as if it wasn't triggered. i'ts a fake EVICT_OK. */
```